builders/manage/sudo/minting.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Minting Native Tokens with Sudo 
+description: Learn how to use the Sudo key to manage your appchain and performing the privileged action of minting additional native tokens of your Tanssi appchain.
+---
+
+# Using Sudo to Mint Native Tokens
+
+## Introduction {: #introduction }
+
+[Sudo](https://paritytech.github.io/polkadot-sdk/master/pallet_sudo/index.html){target=\_blank} is a module that enables privileged runtime calls to be dispatched when called from the Sudo account. Sudo is sometimes colloquially referred to as a superuser or god-like account. This enables you to perform privileged actions in the course of managing your appchain, such as minting new native tokens. 
+
+In this guide, you'll learn how to use Sudo to properly mint new native tokens. This comprehensive guide shows how to check the balance of an existing account prior to assigning it a new balance with Sudo access. 
+
+## Checking Prerequisites {: #checking-prerequisites }
+
+For the examples in this guide, you will need to have the following:
+
+ - A Tanssi Appchain (Snap or Dedicated)
+ - Your Appchain's Sudo account connected to your Appchain's Polkadot.js Apps. You can refer to the [Managing Sudo guide](/builders/manage/sudo/sudo/#configuring-polkadotjs-apps){target=\_blank} for instructions on injecting your Sudo account into Polkadot.js Apps
+
+If you're unsure what your Tanssi Appchain's Sudo account is, you can find it in your [Tanssi Dashboard](https://apps.tanssi.network/){target=\_blank} underneath the **Properties** section.
+
+![Locating your Sudo address on apps.tanssi.network](/images/builders/manage/sudo/minting/minting-1.webp)
```

Comment:
the arrow looks a little bit off. it should be inline with the text its pointing to